proposals/lip-0071.md

```diff
@@ -239,19 +260,20 @@ This function is used to retrieve the block reward for a validator.
 
 ```python
 def getValidatorBlockReward(blockHeader: BlockHeader) -> uint64:
+    height = blockHeader.height
```

Comment:
Suggestion after looking into this more carefully: We could define here `minimumRewardActiveValidators = getMinimumRewardActiveValidators(height)` and then use the constant `minimumRewardActiveValidators` whenever needed (that was my bad to completely remove it).